cli/src/main.rs

```diff
@@ -53,15 +53,35 @@ fn main() -> Result<(), Error> {
                 .help("Determines who is winning. Positive number indicates a white advantage.")
                 .action(clap::ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new("move")
+                .long("move")
+                .short('m')
+                .value_name("move")
+                .help("Make a move from starting square to final square (ex: 'E2:E4'"),
```

Comment:
Does this help message make sense? Maybe just change it to `moves a piece`

Reply:
Fixed